src/index.html

```diff
@@ -10,3 +10,364 @@
     <h1>Catalog</h1>
   </body>
 </html>
+-->
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Catalog</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+    <!-- 
+    <link rel="preconnect" 
+    href="https://fonts.googleapis.com"
+    crossorigin="anonimous"
+    >
+    <link rel="preconnect"  -->
+    <!-- href="https://fonts.gstatic.com"  -->
+    <!-- crossorigin="anonimous" -->
+    <!-- > 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+     -->
+  </head>
+
+  <body class="body">
+    <header class="header">
+      <img class="header__logo" 
+        src="images/logo.png" 
+        alt="logo"
+      >
```

Comment:
```suggestion
      <img
        class="header__logo" 
        src="images/logo.png" 
        alt="logo"
      >
```
if you transfer attributes - 1 per each line